src/main/java/com/xebialabs/restito/server/StubServer.java

```diff
@@ -143,35 +189,35 @@ public StubServer run() {
     }
 
     private SSLContextConfigurator getSslConfig() throws IOException {
-        if(SSLContextConfigurator.DEFAULT_CONFIG.validateConfiguration(true))
-        {
+        SSLContextConfigurator.DEFAULT_CONFIG.retrieve(System.getProperties()); // refresh in case of changes
```

Comment:
I'm wondering, what change requires this now? It's been possible to set up Keystore via properties in the previous version. 🤔

Reply:
It is needed when there are tests running after `shouldBePossibleToSpecifyKeyStoresWithStandardProperties`. There was none, so it worked well. Otherwise, once that test is run, new `StubServer` instance always uses that configuration.